public/js/app.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+let shoppingList = new ShoppingList();
+let content = document.getElementById("content");
+
+let button = document.getElementById("addShoppingListItemButton");
+
+button.addEventListener("click", function addToShoppingList(event) {
+  let title = document.getElementById("title");
+  let description = document.getElementById("description");
+
+  let shoppingListItem = new ShoppingListItem(title.value, description.value);
+
+  shoppingList.addItem(shoppingListItem);
+
+  content.innerHTML = shoppingList.render();
+
+  checkChecked();
+
+  event.preventDefault();
+});
+
+let changeCheckedStatus = function(idx, checkbox) {
+  if (!checkbox.checked === true) {
+    shoppingList.items[idx].uncheck();
+  } else {
+    shoppingList.items[idx].check();
+  }
+};
+
+function checkChecked() {
```

Comment:
Instead of having this function that goes and checks or unchecks each box. You could update the ShoppingListItem's render function to render a input that's already checked.